data/json/npcs/isherwood_farm/NPC_Barry_Isherwood.json

```diff
@@ -77,7 +77,7 @@
     "destination": "dairy_farm_isherwood_W",
     "start": {
       "effect": [ { "u_add_var": "u_have_barry_escape", "type": "general", "context": "meeting", "value": "yes" } ],
-      "assign_mission_target": { "om_terrain": "dairy_farm_isherwood_W", "om_special": "Isherwood Farms", "reveal_radius": 3 }
```

Comment:
I'm pretty sure this should still be here, and is retrieved in <https://github.com/CleverRaven/Cataclysm-DDA/pull/36433/files#diff-0e0da92a1ece3afd4c1e42acd3a3dc51L375>.

As for all the ones struck from the update_mapgen entries: I think the fact that the update_mapgen doesn't use it is an oversight in its implementation rather than useless data on the entries, as without it, it can't differentiate between specials that contain the same overmap terrain (which granted is relatively rare) and thus will target any instance of the omt, but I guess we'll restore the data whenever the implementation gets updated.